.circleci/config.yml

```diff
@@ -499,6 +430,8 @@ workflows:
           requires:
             - build
             - push_acquia
+          refreshDb: "1"
```

Comment:
Nothing big, but does the order in parameter matter?
I'm seeing a different order in multiple places.